packages/dev/docs/pages/react-aria/index.mdx

```diff
@@ -29,7 +32,11 @@ description: A library of React Hooks that provides accessible UI primitives for
 
 A library of React Hooks that provides accessible UI primitives for your design system.
 
-<a href="getting-started.html" style={{fontSize: 18}}>Get started</a>
+<p className={classNames(typographyStyles['spectrum-Body2'], styles.homeLinks)}>
+  <a href="getting-started.html">Get started</a>
+  <span aria-hidden="true">•</span>
+  <a href="https://github.com/adobe/react-spectrum">GitHub</a>
```

Comment:
should these go into the packages themselves? https://github.com/adobe/react-spectrum/tree/main/packages/%40react-aria

Reply:
Seems better to go to the root where we have the readme and stuff